demo_drivers/reaction_diffusion/one_d_act_inhibit/validate.sh

```diff
@@ -6,24 +6,25 @@ OOMPH_ROOT_DIR=$(make -s --no-print-directory print-top_builddir)
 #Set the number of tests to be checked
 NUM_TESTS=1
 
+# Setup validation directory
+#---------------------------
+touch Validation
+rm -r -f Validation
+mkdir Validation
+
 # Disable the self-test on macOS as it causes a mysterious segmentation fault.
-# To do this, we return an exit code of 0 so it looks like the test passed
+# To do this, we provide a dummy OK in Validation/validation.log so return an exit code of 0 so it looks like the test passed. Make
+# sure to create a
```

Comment:
Does this comment end somewhat prematurely?

Reply:
It does indeed, thanks for spotting this! I'll change it now.